### ISSUE TITLE
Error gettign stream URL: HTTP Error 402: Payment Required

### ISSUE THREAD

**Issue by xarziv** [mps-youtube/pafy#106]
I know there is an issue below that, however i checked my own browser and there is no captcha. I tried using pafy on my own desktop and it loaded perfectly. When i try on my website it doesn't work. Tells me that payment is required. 

So, i made a Python file called pafy.py (put the pafy.py code in it), then made __init__.py (had __init__.py code inside it)

Then inside the pafy.py file i replaced the two places where we can put our api, with my own API, and i still get the error. I checked my quota and have like 49,000 left. It only gets fixed when i change my website IP. However shouldn't it work even though i sued my own API and quota wasn't filled yet?

**Comment by ids1024** [comment]
It seems the IP is blocked by youtube for some reason.

Duplicate of #98.
